centreon/src/Core/ResourceAccess/Application/UseCase/DeleteRules/DeleteRulesStatusResponse.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\ResourceAccess\Application\UseCase\DeleteRules;
+
+use Core\ResourceAccess\Domain\Model\ResponseCode;
+
+final class DeleteRulesStatusResponse
+{
+    public int $id = 0;
+
+    public ResponseCode $status = ResponseCode::OK;
```

Comment:
Is it possible to use the Response constant instead of an enum (to reduce base code to maintain) ? 
https://github.com/symfony/symfony/blob/7.0/src/Symfony/Component/HttpFoundation/Response.php#L28

Reply:
On this one IDK. as Symfony Response entity is dedicated to the Infrastructure (in our case).
Furthermore the UseCase should be independent of the Framework that is why there is this ResponseCode DTO.